snapshot/git/gitdb/bundlestore.go

```diff
@@ -126,7 +127,10 @@ func (b *bundlestoreBackend) uploadLocalSnapshot(s *localSnapshot, db *DB) (sn s
 		return nil, err
 	}
 
-	bundleName := makeBundleName(s.sha)
+	bundleSnap := &bundlestoreSnapshot{sha: s.sha, kind: s.Kind(), bundleKey: s.sha, streamName: streamName}
+
+	//bundleName := makeBundleName(s.sha)
+	bundleName := string(bundleSnap.ID()) //TODO: why have a separate snapshotId and bundleName? It's confusing to track...
```

Comment:
Because we may end up having two Snapshots in the same Bundle; Cf. the comment below above the return.